lambda/src/main/resources/db/migration/V71__add_ui_element_descriptions_to_FileProperty_table.sql

```diff
@@ -0,0 +1,19 @@
+-- This script adds description information used for fields on the 
```

Comment:
The migration version number in the file name will need to be updated. The latest number is `V75`. So this will need to be changed, based on the other migration PRs.